javav2/example_code/ssm/src/main/java/com/example/scenario/SSMScenario.java

```diff
@@ -0,0 +1,567 @@
+import software.amazon.awssdk.services.ssm.model.OpsItemFilterOperator;
+import software.amazon.awssdk.services.ssm.model.OpsItemStatus;
+import software.amazon.awssdk.services.ssm.model.OpsItemSummary;
+import software.amazon.awssdk.services.ssm.model.SendCommandRequest;
+import software.amazon.awssdk.services.ssm.model.SendCommandResponse;
+import software.amazon.awssdk.services.ssm.model.SsmException;
+import software.amazon.awssdk.services.ssm.model.UpdateMaintenanceWindowRequest;
+import software.amazon.awssdk.services.ssm.model.UpdateOpsItemRequest;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Before running this Java V2 code example, set up your development
+ * environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html
+ *
+ * AWS Systems Manager Agent (SSM Agent) is Amazon software that runs on Amazon Elastic Compute Cloud (Amazon EC2) instances,
+ * edge devices, on-premises servers, and virtual machines (VMs). SSM Agent makes it possible for Systems Manager to
+ * update, manage, and configure these resources.
+ * To successfully run this getting started scenario, the EC2 instance must have an SSM Agent.
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html.
+ *
+ *
+ * This Java program performs these tasks:
+ * 1. Creates an AWS Systems Manager maintenance window with a default name or a user-provided name.
+ * 2. Modifies the maintenance window schedule.
+ * 3. Creates a Systems Manager document with a default name or a user-provided name.
+ * 4. Sends a command to a specified EC2 instance using the created Systems Manager document and displays the time when the command was invoked.
+ * 5. Creates a Systems Manager OpsItem with a predefined title, source, category, and severity.
+ * 6. Updates and resolves the created OpsItem.
+ * 7. Deletes the Systems Manager maintenance window, OpsItem, and document.
+ */
+
+public class SSMScenario {
+    public static final String DASHES = new String(new char[80]).replace("\0", "-");
+    public static void main(String[] args) throws InterruptedException {
+        String usage = """
+            Usage:
+              <instanceId> <title> <source> <category> <severity>
+      
+            Where:
+                instanceId - The Amazon EC2 Linux/UNIX instance Id that AWS Systems Manager uses (ie, i-0149338494ed95f06). 
+                title - The title of the parameter (default is Disk Space Alert).
+                source - The source of the parameter (default is EC2).
+                category - The category of the parameter (default is Performance).
```

Comment:
category only allows certain values. You should document it.

"Availability", "Cost", "Performance", "Recovery", "Security"